### PR TITLE
Add INT8 OneDNN documentation

### PR DESCRIPTION
**Pull request by wozna**
This PR  adds INT8 OneDNN documentation about PTQ and QAT approaches.

### CODE REVIEW

File: docs/design/mkldnn/int8/QAT/README.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+   ```... → input1 → conv2d → output1 → batch_norm → output2 → relu → output3 → ...```
+   and we want to quantize the `conv2d` op, then after applying FP32 optimizations the sequence will become
+   ```... → input1 → conv2d → output3 → ...```
+   and the quantization scales have to be collected for the `input1` and `outpu3` tensors in the Quant model.
+2. Quantization of the following operators is supported: `conv2d`, `depthwise_conv2d`, `mul`, `fc`, `matmul`, `pool2d`, `reshape2`, `transpose2`, `concat`.
+3. The longest sequence of consecutive quantizable operators in the model, the biggest performance boost can be achieved through quantization:
+   ```... → conv2d → conv2d → pool2d → conv2d → conv2d → ...```
+   Quantizing single operator separated from other quantizable operators can give no performance benefits or even slow down the inference:
+   ```... → swish → fc → softmax → ...`
```
Comment (sfraczek) [inline docs/design/mkldnn/int8/QAT/README.md]:
```suggestion
   ```... → swish → fc → softmax → ...```
```

**Comment by wozna** [comment]
@jiangjiajun Could you start your review?

**Review by sfraczek — APPROVED** [review]
LGTM

File: docs/design/mkldnn/int8/PTQ/README.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
+# INT8 OneDNN post-training quantization
+
+This document is based on a previously created document [post-training quantization](https://github.com/PaddlePaddle/Paddle/blob/develop/paddle/fluid/inference/tests/api/int8_mkldnn_quantization.md) and it describes how to use Paddle inference Engine to convert the FP32 models to INT8 models using INT8 MKL-DNN post-training quantization. We provide the instructions on enabling INT8 MKL-DNN quantization in Paddle inference and show the accuracy and performance results of the quantized models, including 7 image classification models: GoogleNet, MobileNet-V1, MobileNet-V2, ResNet-101, ResNet-50, VGG16, VGG19, and 1 object detection model Mobilenet-SSD.
+
+## 0. Install PaddlePaddle
+
+Follow PaddlePaddle [installation instruction](https://github.com/PaddlePaddle/models/tree/develop/fluid/PaddleCV/image_classification#installation) to install PaddlePaddle. If you build PaddlePaddle yourself, please use the following cmake arguments.
```
Comment (yeliang2258) [inline docs/design/mkldnn/int8/PTQ/README.md]: The path here is invalid and cannot be opened.

Comment (wozna) [reply docs/design/mkldnn/int8/PTQ/README.md]: You are right, thanks.

**Comment by jczaja** [comment]
@yaomichael Please help this one to be reviewed and merged

File: docs/design/mkldnn/int8/PTQ/index.html [hunk 3]
```diff
@@ -0,0 +1,229 @@
+
```
Comment (qingqing01) [inline docs/design/mkldnn/int8/PTQ/index.html:1]: It seems there is no need to upload html file

**Review by qingqing01 — COMMENTED** [review]
LGTM.  But it seems there is no need to upload HTML file.

**Comment by wozna** [comment]
> LGTM. But it seems there is no need to upload HTML file.

I removed html files and update some information. Could you please check one more time?

**Review by qingqing01 — APPROVED** [review]
